server/sonar-server/src/main/java/org/sonar/server/issue/index/IssueIndex.java

```diff
@@ -360,6 +365,7 @@ public SearchResponse search(IssueQuery query, SearchOptions options) {
 
     configureStickyFacets(query, options, filters, esQuery, requestBuilder);
     requestBuilder.setFetchSource(false);
+    System.out.println(requestBuilder);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS106)